go.mod

```diff
@@ -4,6 +4,11 @@ go 1.24.1
 
 toolchain go1.24.2
 
+require github.com/amikos-tech/chroma-go v0.2.2
```

Comment:
https://github.com/amikos-tech/chroma-go is maintained by a member of the Chroma team.